apps/web/composables/useProductReviews/useProductReviews.ts

```diff
@@ -75,7 +76,9 @@ export const useProductReviews: UseProductReviews = (itemId: number, productVari
 
     const { send } = useNotification();
     const { $i18n } = useNuxtApp();
-    const { data, error } = await useAsyncData(() => useSdk().plentysystems.doReview(params));
+    const { data, error } = await useAsyncData(`${JSON.stringify(params)}`, () =>
```

Comment:
just remove the async data here. This is a client side action that does not benefit of useAsyncData.